backend/demo/src/main/java/com/snacks/demo/dto/UserDto.java

```diff
@@ -0,0 +1,21 @@
+package com.snacks.demo.dto;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
```suggestion
@Data
```
`@Data` 어노테이션을 쓸 수 있을까요?